cypress/integration/errorPagePersian.js

```diff
@@ -8,7 +8,7 @@ import {
 import { testResponseCode } from '../support/metaTestHelper';
 import persian from '../../src/app/lib/config/services/persian';
 
-describeForLocalOnly('Article Body Tests', () => {
+describeForLocalAndTest('Article Body Tests', () => {
```

Comment:
This falls back to the Mozart page on Test currently and it doesn't have the same error message, so it will fail. Once these issues have been addressed, this change can be made https://github.com/bbc/simorgh/issues/1952 https://github.com/bbc/simorgh-infrastructure/issues/347
```suggestion
describeForLocalOnly('Article Body Tests', () => {
```